README.md

```diff
@@ -1 +1,29 @@
-# Task-List-App
\ No newline at end of file
+# Task-List-App
+## Project Name
+"Todo List App"  is a project design to add and remove task to/from the web page, its a solo project set up by Microverse as a core requirement the code was broken into smaller fragments called module. 
+this is the basic structure for project
+
+## Built With
+The major used for this project are : 
+HTML,
+Javascript 
+
+## Technologies used
+The amazing tecnologies for this project are:
+Linters,
+git
+ ## How to  make a copy of this project(clone):
+ Visit our github url listed below
+ search project(repo name)
+ use git fork and pull to ur locall machine or download the zip code .. 
```

Comment:
- [ ] [OPTIONAL] Your readme file looks descriptive and professional but please Update your readme file with explicit steps on how to clone and run your tests locally, as this will be of help to non-technical users. :smile: